tests/api_connexion/schemas/test_xcom_schema.py

```diff
@@ -0,0 +1,193 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import unittest
+
+from sqlalchemy import or_
+
+from airflow.api_connexion.schemas.xcom_schema import (
+    XComCollection, xcom_collection_item_schema, xcom_collection_schema, xcom_schema,
+)
+from airflow.models import XCom
+from airflow.utils import timezone
+from airflow.utils.session import create_session, provide_session
+
+
+class TestXComCollectionItemSchema(unittest.TestCase):
+
+    def setUp(self) -> None:
+        self.now = timezone.utcnow()
+        with create_session() as session:
+            session.query(XCom).delete()
+
+    @provide_session
+    def test_serialize(self, session):
+        xcom_model = XCom(
+            key='test_key',
+            timestamp=self.now,
```

Comment:
If you use a date string above, you can parse the time with timezone.parse(timestring)